include/swift/Serialization/SerializationOptions.h

```diff
@@ -24,144 +24,144 @@
 
 namespace swift {
 
-  class SerializationOptions {
-    SerializationOptions(const SerializationOptions &) = delete;
-    void operator=(const SerializationOptions &) = delete;
+class SerializationOptions {
```

Comment:
Nit: I assume clang-format fixed the formatting here for the whole class. It probably needed to happen eventually but it would be nice for it all to be fixed in a separate commit so that it's clearer what's happening in the PR

Reply:
Done